lapce-ui/src/explorer.rs

```diff
@@ -57,15 +62,17 @@ fn paint_single_file_node_item(
     };
 
     if let Some(background) = background {
-        ctx.fill(
-            Rect::ZERO
-                .with_origin(Point::new(
-                    0.0,
-                    current as f64 * line_height - line_height,
-                ))
-                .with_size(Size::new(width, line_height)),
-            config.get_color_unchecked(background),
-        );
+        ctx.fill(rect, config.get_color_unchecked(background));
+    }
+
+    let border = if Some(current) == focused {
+        Some(LapceTheme::EDITOR_CARET)
```

Comment:
Wouldn't `PANEL_CURRENT` be more appropriate here, or a new color? Reusing the caret as a border color seems a bit weird to me.